.gitignore

```diff
@@ -8,3 +8,16 @@
 *.pyc
 *~
 /cbuildbot_package.list
+
+# Flatcar SDK tarballs
+*.tar.bz2
+
+# SDK container env passing helpers
+sdk_container/.env
+sdk_container/.sdkenv
+
+# build cache / artefacts directories
+__build__/
+sdk_container/.cache
+sdk_container/.config
+sdk_container/.lock
```

Comment:
This is probably not necessary any more.

Reply:
Nice catch, I'll remove `.lock`.